__tests__/app.test.js

```diff
@@ -5,4 +5,48 @@ const { string } = require("pg-format");
 
 afterAll(() => {
   if (db.end) db.end();
+});
+
+describe('GET /api', () => {
+    test('should return an object in the body', () => {
+        return request(app)
+        .get("/api")
+        .expect(200)
+        .then(({ body }) => {
+            expect(body).toBeInstanceOf(Object);
+            expect(Object.keys(body)).toEqual(["message"]);
+            expect(body.message).toBe("ok");
+          });
+    });
+});
+
+describe('GET /api for non existant route', () => {
+    test('should respond with a 404 when provided with a non existant route', () => {
+      return request(app)
+        .get("/api/non-existant-route")
+        .expect(404)
+        .then(({ body }) => {
+          expect(body).toEqual({msg: 'Path Not Found'});
+        })
+    });
+  });
+
+
+describe('GET /api/categories', () => {
+    test('should return an object with the key of categories and an array of the category objects', () => {
+        return request(app)
+        .get("/api/categories")
+        .expect(200)
+        .then(({ body }) => {
+            expect(body).toBeInstanceOf(Object);
```

Comment:
Checking body is an instance of Object and checking it has a key of categories in line 42 are both redundant tests as you restructure from it on line 43 using the key 'categories' (which kills two birds with one stone)

you can happily remove line 41 and 42